walletnode/services/artworkregister/task.go

```diff
@@ -213,7 +221,7 @@ func (task *Task) waitTxidValid(ctx context.Context, txID string, expectedConfir
 			} else {
 				return nil
 			}
-		case <-time.After(timeout):
+		case <-timeoutCh:
```

Comment:
Thank you. Good catch here because `time.After` is rearmed every time the select run again.